Jenkinsfile

```diff
@@ -16,6 +16,7 @@ pipeline {
     DOCKER_SECRET = 'secret/apm-team/ci/docker-registry/prod'
     DOCKER_REGISTRY = 'docker.elastic.co'
     DOCKER_IMAGE = "${env.DOCKER_REGISTRY}/observability-ci/apm-server"
+    ONLY_DOCS = "false"
```

Comment:
Nit: This seems to be a string whereas on L82 we set it to a bool when we use the value from `isGitRegionMatch`. Then on L106 it seems like we eval it as a string again.

Reply:
Environment vars are stored as Strings no matter the input type.